test/DynamoCoreTests/DynamoModelTestBase.cs

```diff
@@ -91,6 +98,20 @@ protected virtual void GetLibrariesToPreload(List<string> libraries)
             // derived class.
         }
 
+        protected virtual string GetUserUserDataRootFolder()
+        {
+            // Override in derived classed to provide a custom
+            // UserAppDataRootFolder
+            return string.Empty;
+        }
+
+        protected virtual string GetCommonDataRootFolder()
+        {
+            // Override in derived classed to provide a custom
+            // CommonAppDataRootFolder
+            return string.Empty;
+        }
+
```

Comment:
I'm assuming passing String.Empty will cause the default to be set in PathResolver.  Is that right?  We might want to clarify that point.